packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -96,6 +96,27 @@ export class ICXOrderBook {
       'bignumber'
     )
   }
+
+  /**
+   * Returns information about orders or fillorders based on ICXListOrderOptions passed
+   *
+   * @param {ICXListOrderOptions} options
+   * @param {string}  [options.token] Token asset
+   * @param {string}  [options.chain] Chain asset
+   * @param {string}  [options.orderTx] Order txid to list all offers for this order
+   * @param {number}  [options.limit] Maximum number of orders to return (default: 50)
+   * @param {boolean} [options.closed] Display closed orders (default: false)
+   * @return {Promise<Record<string, ICXOrderInfo | ICXOfferInfo>>} Object indluding details of the transaction.
```

Comment:
is `overload` method possible to implement here?
what's condition will return ICXOrderInfo but ICXOfferInfo??

Reply:
if you don't specify `ICXListOrderOptions.orderTx` it will return orders in the system. i.e. `ICXOrderInfo`
if specified, only the offers (`ICXOfferInfo`) for that specific order will be returned.